<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Santosh-PC" ignored="0" name="BaningApp Test " tests="7" failures="4" timestamp="01 Sep 2020 10:42:46 GMT" time="131.633" errors="0">
  <testcase name="loginTest" time="2.821" classname="com.inetbanking.testCases.TC_LoginTest_001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginTest_001.loginTest(TC_LoginTest_001.java:37)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginDDT" time="12.11" classname="com.inetbanking.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="5.856" classname="com.inetbanking.testCases.TC_LoginDDT_002">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:31)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="5.63" classname="com.inetbanking.testCases.TC_LoginDDT_002">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:31)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="12.232" classname="com.inetbanking.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="6.16" classname="com.inetbanking.testCases.TC_LoginDDT_002">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:31)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="addNewCustomer" time="19.542" classname="com.inetbanking.testCases.TC_AddCustomerTest_003"/>
</testsuite> <!-- BaningApp Test  -->
